core/controllers/profile.py

```diff
@@ -76,6 +77,8 @@ def get(self, username):
         user_settings = user_services.get_user_settings_from_username(username)
         if not user_settings:
             raise self.PageNotFoundException
+        else:
```

Comment:
No need else -- just deindent the following line by 4. The pattern is: if special condition arises, raise; if another special condition arises, raise; after that assume no more special conditions apply.